cap_client/cap_api.py

```
@@ -338,6 +338,16 @@ def upload(self, pid=None, filepath=None, yes=False, output_filename=None):
             method='put',
         )
 
+    def upload_docker_img(self, pid=None, img_name=None):
+        """Uploads docker image."""
+        from subprocess import check_call, CalledProcessError
+        try:
+            check_call(["docker", "save", "-o", "{}.tar".format(img_name), img_name])
+            self.upload(pid=pid, filepath=img_name, output_filename=img_name)
+            check_call(["rm", "{}.tar".format(img_name)])
+        except CalledProcessError:
```

don't hide exceptions ;)